examples/bri-3/src/bri/workgroup/workflows/models/workflow.ts

```diff
@@ -1,11 +1,48 @@
 import { Workstep } from '../../worksteps/models/workstep';
 
 export class Workflow {
-  private id: string; // TODO: Add uuid after #491
-  private worksteps: Workstep[]; //TODO enforce workstep causal connection through collection order
+  private _id: string; // TODO: Add uuid after #491
+  private _name: string;
+  private _worksteps: Workstep[]; //TODO enforce workstep causal connection through collection order
```

Comment:
@Kasshern might be easiest to enforce order through a binary sparse merkle tree of worksteps organized into a workflow. just a thought.

Reply:
@Therecanbeonlyone1969 Will create a separate issue corresponding to this TODO and add your advice.